test/test_cpp_extensions_aot.py

```diff
@@ -367,5 +367,13 @@ def f(a: bool, b: bool):
         self.assertIn("torch_library::logical_and", str(s.graph))
 
 
+class TestDeviceBackendAutoload(common.TestCase):
+    def test_autoload(self):
```

Comment:
Not sure if I understand it correctly but when the cpp extension is explicitly imported from this test file, not autoloaded implicitly. Is that correct?

Reply:
The reason for putting this test case here is that I want to reuse the existing modules in `test/cpp_extension/setup.py`.